web/sites/default/config/webform.webform._template_standard_webform.yml

```diff
@@ -631,7 +644,7 @@ handlers:
     settings:
       states:
         - completed
-      to_mail: gregoryscottclapp@gmail.com
+      to_mail: powr@portlandoregon.gov
```

Comment:
I wonder if we want to make this 311@portlandoregon.gov in a follow up so that that's our "default". 🤔

Reply:
Oh, good catch. I'll update it in my Location widget bug fix story.